api/pkg/cli/cmd/get/get.go

```diff
@@ -97,10 +99,16 @@ func (opts *options) run() error {
 		return err
 	}
 
-	hubClient := opts.cli.Hub()
+	opts.hubClient = opts.cli.Hub()
+	var err error
 
-	resource := hubClient.GetResource(hub.ResourceOption{
-		Name:    opts.name(),
+	name, err := opts.GetResourceInfo()
```

Comment:
can we keep similar steps as `info` command?
it was nice instead of keeping inside `opts.GetResourceInfo()`

Reply:
Yes will update the pr once info gets in